1000$

```diff
@@ -2,61 +2,82 @@
 #include "balance.cpp"
 #include "randomEngine.cpp"
 
+struct Character{
+    int humanNumber1;
+    int humanNumber2;
+    int pcValue1;
+    int pcValue2;
+};
+
 class NumberGame{
-    int computerChoice;
     int randNumber;
-    int humanChoice;
     Balance balance = Balance(1000, 1000);
-    RandomEngine randomService;
+    RandomEngine randomEngine;
 public:
     void run(){
-        humanChoice = getPlayerInput();
-        computerChoice = calcComputerTurn(humanChoice);
-        randNumber = randomService.getRandomNumber();
-            int absComputer = std::abs(computerChoice - randNumber);
-            int absHuman = std::abs(humanChoice - randNumber);
+        while(balance.getHumanScoreBalance() >= 0 || balance.getPcScoreBalance() >= 0) {
```

Comment:
has to be more then 100$

Could we more this logic to balance class and create method
balanc.isAvalibleForGame() - Boolean